### PR TITLE
Nisarg

### PR DESCRIPTION
**Pull request by Nisarg1206**

### CODE REVIEW

File: lib/WelcomeScreen.dart [hunk 1]
```diff
@@ -0,0 +1,221 @@
+import 'dart:js';
+import 'dart:ui';
+
+import 'package:flutter/cupertino.dart';
```
Comment (RitwikSingh28) [inline lib/WelcomeScreen.dart:4]: Please remove this import as it is already included in the material.dart import by default.

File: lib/WelcomeScreen.dart [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+import 'package:flutter/material.dart';
+
+void main() {
```
Comment (RitwikSingh28) [inline lib/WelcomeScreen.dart:7]: Do not use main function here... Kindly remove it

File: lib/WelcomeScreen.dart [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+  runApp(MyApp());
+}
+
+class MyApp extends StatefulWidget {
```
Comment (RitwikSingh28) [inline lib/WelcomeScreen.dart:11]: Rename your widget to WelcomeScreen

**Comment by SrijanShovit** [comment]
The approprite changes were not done for long time so closing this PR with invlid mark.

**Pull request closed without merging**